### PR TITLE
Hw 18

### PR DESCRIPTION
**Pull request by TarasHryts**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -43,6 +59,43 @@
                 </configuration>
 
             </plugin>
+
+            <!--            <plugin>-->
```
Comment (Sofasmile) [inline pom.xml]: remember about comment from previous hw

File: src/main/java/mate/academy/shop/anotation/AnnotatedClassMap.java [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
 import mate.academy.shop.service.OrderService;
 import mate.academy.shop.service.UserService;
-import mate.academy.shop.factory.Factory;
-
-import java.util.HashMap;
-import java.util.Map;
+import org.apache.log4j.Logger;
 
 public class AnnotatedClassMap {
+    final static Logger logger = Logger.getLogger(FileReader.class);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/shop/anotation/AnnotatedClassMap.java]: the same

File: src/main/java/mate/academy/shop/web/MainServlet.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package mate.academy.shop.web;
+
+public class MainServlet {
```
Comment (Sofasmile) [inline src/main/java/mate/academy/shop/web/MainServlet.java:3]: do we need this class?

File: pom.xml (hunk 1 above)
Comment (TarasHryts) [reply pom.xml]: already pushed changes from the previous task

File: src/main/java/mate/academy/shop/anotation/AnnotatedClassMap.java (hunk 2 above)
Comment (TarasHryts) [reply src/main/java/mate/academy/shop/anotation/AnnotatedClassMap.java]: already pushed changes from the previous task

File: src/main/java/mate/academy/shop/web/MainServlet.java (hunk 3 above)
Comment (TarasHryts) [reply src/main/java/mate/academy/shop/web/MainServlet.java:3]: Bohdan told that we will need it later

**Review by boroda4436 — APPROVED** [review]

**Pull request closed without merging**